ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/CommandCompletionsReader.scala

```diff
@@ -35,13 +36,13 @@ trait CommandCompletionsReader[LedgerOffset] {
     *
     * TODO Drop the LedgerOffset from the source when we replace the Dispatcher mechanism
     *
-    * @param startInclusive starting offset inclusive
-    * @param endExclusive   ending offset exclusive
+    * @param startExclusive starting offset inclusive
+    * @param endInclusive   ending offset exclusive
```

Comment:
I was about to suggest to fix these, but given that the comment doesn't add any information it's probably better to just delete it.:wink:

Reply:
Will delete. See 949dc9225 in #4953.